pkgs/development/libraries/openexrid/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, unzip, re2, openfx, zlib, ilmbase, mesa, openexr }:
+
+stdenv.mkDerivation rec
+{
+  name = "openexrid-${version}";
+  version = "2017-09-17";
+
+  src = fetchurl {
+    url = "https://github.com/MercenariesEngineering/openexrid/archive/bec0081548a096f9bcdd1504970c96264b0fc050.zip";
```

Comment:
These fetches may be better with fetchFromGitHub

Reply:
And actually it was written inside `CONTRIBUTING.md`. Sorry, I'm fixing that.